SingularityBase/src/main/java/com/hubspot/singularity/SingularityS3UploaderFile.java

```diff
@@ -57,6 +60,7 @@ public SingularityS3UploaderFile(
     this.applyS3StorageClassAfterBytes = applyS3StorageClassAfterBytes;
     this.checkSubdirectories = checkSubdirectories.orElse(false);
     this.compressBeforeUpload = compressBeforeUpload.orElse(false);
+    this.checkIfOpen = checkIfOpen.orElse(false);
```

Comment:
should this be `true` if the default is true?

Reply:
woof, good catch